community/youtube-dl/APKBUILD

```diff
@@ -20,7 +20,7 @@ source="https://github.com/rg3/$pkgname/releases/download/$pkgver/$pkgname-$pkgv
 builddir="$srcdir/$pkgname"
 
 prepare() {
-	cd "$builddir"
+	default_prepare
 	sed -i setup.py \
```

Comment:
Yes we need default_prepare, but im not sure I like having to depend on a function to change directory.
Instead I would use full path to setup.py or change directory to it.

Reply:
All set.